pkg/flare/archive.go

```diff
@@ -371,6 +376,23 @@ func zipExpVar(tempDir, hostname string) error {
 	return err
 }
 
+func zipSystemProbeStats(tempDir, hostname string) error {
+	sysProbeStats := status.GetSystemProbeStats()
+	sysProbeFile := filepath.Join(tempDir, hostname, "expvar", "system-probe")
+	sysProbeWriter, err := newRedactingWriter(sysProbeFile, os.ModePerm, true)
```

Comment:
Do we store secrets in expvar ? Because expvar a fully open on the localhost, no ?

Reply:
We don't in system probe - honestly, I was just following the behavior in the zipExpVar function which used a RedactingWriter. 

If you need/want me to change this to a general writer, let me know @hush-hush!

Reply:
I don't think it hurts, to let's keep it. I just wanted to double check that we didn't had a security issue on our hands !